tools/slack-release-notifier/Dockerfile

```
@@ -0,0 +1,19 @@
+FROM node:12-alpine
+
+COPY ./package.json /slack-release-notifier/package.json
+COPY ./package-lock.json /slack-release-notifier/package-lock.json
+COPY ./tsconfig.json /slack-release-notifier/tsconfig.json
```

You can combine the copy into one layer makes the docker image smaller because it has fewer layers

```suggestion
COPY \
  ./package.json  \
  ./package-lock.json \
  ./tsconfig.json \
  /slack-release-notifier
```